include/ensmallen_bits/cne/cne_impl.hpp

```diff
@@ -70,8 +68,10 @@ double CNE::Optimize(DecomposableFunctionType& function, arma::mat& iterate)
         "children. Increase population size.");
   }
 
-  // Set the population size and fill random values [0,1].
-  population = arma::randu(iterate.n_rows, iterate.n_cols, populationSize);
+  // Generate the population based on a Gaussian disribution around the given
```

Comment:
Oops, tiny typo, this should be `distribution`. :)

Reply:
Just when i thought i had caught everything xD